depclean-maven-plugin/src/main/java/se/kth/depclean/DepCleanMojo.java

```diff
@@ -375,7 +375,7 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     String projectJar = project.getArtifactId() + "-" + project.getVersion() + ".jar";
     long projectSize = FileUtils.sizeOf(new File(project.getBuild().getOutputDirectory()));
     sizeOfDependencies.put(projectJar, projectSize);
-    if (Files.exists(Path.of(
+    if (Files.exists(Paths.get(
```

Comment:
@cesarsotovalero it seems Path.of is not supported by JDK8, changed to Paths.get.